### PR TITLE
Move extension in own folder

### PR DESCRIPTION
**Pull request by ivy-cst**
So that the root package.json does not conflict with the extension package.json

### CODE REVIEW

**Comment by ivy-cst** [comment]
Please check carefully!

File: .vscode/tasks.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
-            "presentation": {
-                "reveal": "always",
-                "panel": "dedicated"
-              }
-          }
-    ]
+  "version": "2.0.0",
+  "tasks": [
+    {
+      "label": "clean & build extension",
```
Comment (ivy-lmu) [inline .vscode/tasks.json:5]: also update the `preLaunchTask` in launch.json that references this label

File: playwright/tests/workspaces/default/.project [hunk 2]
```diff
@@ -46,4 +46,15 @@
 		<nature>org.eclipse.wst.common.project.facet.core.nature</nature>
 		<nature>org.eclipse.wst.jsdt.core.jsNature</nature>
 	</natures>
-</projectDescription>
\ No newline at end of file
+	<filteredResources>
```
Comment (ivy-lmu) [inline playwright/tests/workspaces/default/.project:49]: notice this auto generated stuff here

**Review by ivy-lmu — APPROVED** [review]
great 👍  thanks

**Pull request merged**